src/components/Text.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { css, cx } from "emotion";
+import { colors } from "../theme";
+import { TextProps } from "./typings/Text";
+
+const Text: React.FunctionComponent<TextProps> = ({
+  typography,
+  color = colors.gray.base,
```

Comment:
Shouldn't the default color be `colors.gray.darker` as it is used more frequently?

Reply:
done